File: Work/Strings/Practice.py
#Ryan R, Name Decorator

name = input("What is your first name?\n").lower().strip().capitalize()
print(f"Hello {name}, welcome to the name decorator.")
decor = input("How would you like to decorate your name? (Just type the number) \n1. xXNameXx\n2. (:(: Name :) :)\n3. !?Name!?\n Number:") .strip()

dec1 = "xX"
dec2 = "Xx"
dec3 = "(:(:"
dec4 = ":):)"
dec5 = "!?"
if(decor == "1"): print(dec1+name+dec2)
if(decor == "2"): print(dec3+name+dec4)
if(decor == "3"): print(dec5+name+dec5)

File: Work/Strings/Practice.c
//Ryan Rowbury, Name Decorator
#include <stdio.h>
#include <string.h>


char name[20];
char dec1[20]= "xX";
char dec2[20]= "Xx";

int main(void){
    printf("Please tell me your first name\n");
    scanf("%s", name);
    strcat(dec1, name);
    strcat(dec1, dec2);
    printf("This is your Decorated name, %s", dec1);
    return(0);
}